README.md

```diff
@@ -141,13 +125,9 @@ vim.api.nvim_create_autocmd("User", {
 vim.g.rustaceanvim = {
 	server = {
 		auto_attach = function(bufnr)
-			vim.api.nvim_create_autocmd("User", {
-				pattern = { "DirenvLoaded", "DirenvNotFound" },
-				callback = function()
+            M.on_direnv_finished({ once = true }, function ()
```

Comment:
I don't actually use `rustaceanvim`, so not sure if this works - I just tried to transpose over the opts from the previous example.

Reply:
See comments above.

Also, when passing no `opts["filetype"]` like here, doesn't the callback end up being a nop because of the `opts["filetype"] ~= nil` check?
Should it pass `filetype = {"rust"}` ?

Maybe `on_direnv_finished()` doesn't need to be in the plugin but could be given as an example in the README? But I don't feel strongly about it

Reply:
Good catch, have fixed the `~= nil` condition - thanks.